src/Robo/Commands/Setup/ConfigCommand.php

```diff
@@ -35,6 +35,13 @@ public function update() {
       throw new BltException("Failed to execute database updates!");
     }
 
+    $task->drush("cache-rebuild");
+    $result = $task->run();
+
+    if (!$result->wasSuccessful()) {
+      throw new BltException("Failed to execute cache-rebuild!");
```

Comment:
Let's also add a comment explaining why this is here. I can just see someone opening a PR in 6 months to remove this because "updb already runs a cache clear!".

Reply:
mostly I added it there to make sure that $result was used. I didn't want an unused variable, and this seemed to be the standard. open to other thoughts though? i agree, a cr seems a silly thing to throw on.

Reply:
You could just `$task->run();` without capturing the result. We don't really care about the result for a cache clear.

Reply:
Or print a warning instead of throwing an exception.